test/metabase/test/data/interface.clj

```diff
@@ -668,5 +667,5 @@
    (or (db-test-env-var driver env-var default)
        (throw (Exception. (format "In order to test %s, you must specify the env var MB_%s_TEST_%s."
                                   (name driver)
-                                  (str/upper-case (name driver))
+                                  (str/upper-case (str/replace (name driver) #"-" "_"))
```

Comment:
Fixed error message if you tried to require an env var in order to test `:bigquery-cloud-sdk` -- it would warn that "`MB_BIQUERY-CLOUD-SDK_TEST_ACCOUNT_ID` is required to test `:bigquery-cloud-sdk`" or whatever